Makefile

```diff
@@ -72,6 +74,9 @@ submodules:
 	@git submodule sync
 	@git submodule update --init --recursive
 
+go-integration:
+	GO_TEST_FLAGS="-timeout 1h -count=1 -v" GO_TAGS=integration $(MAKE) go.test.integration
```

Comment:
How does `go.test.integration` work?

Reply:
It basically just calls `go test` on the subdir defined at top of `Makefile`. It is more opaque than I would like for it to be, and I would be happy to call `go test` with the integration tags directly if that is more clear :+1:

Reply:
Yeah that sounds great. Thanks!